client-internal/src/main/java/org/terracotta/angela/client/support/junit/AngelaRule.java

```diff
@@ -88,27 +86,14 @@ public ConfigurationContext configure(ConfigurationContext configuration) {
 
   @Override
   protected void before(Description description) throws Throwable {
-    final int nodePortCount = computeNodePortCount();
 
-    AngelaOrchestrator angelaOrchestrator = angelaOrchestratorSupplier.get();
-
-    PortAllocator.PortReservation nodePortReservation = angelaOrchestrator.getPortAllocator().reserve(nodePortCount);
-
-    // assign generated ports to nodes
-    for (TerracottaServer node : configuration.tsa().getTopology().getServers()) {
-      if (node.getTsaPort() <= 0) {
-        node.tsaPort(nodePortReservation.next());
-      }
-      if (node.getTsaGroupPort() <= 0) {
-        node.tsaGroupPort(nodePortReservation.next());
-      }
-    }
```

Comment:
port allocation which was done before for dynamic config in the angela rule is now done at the creation time of the cluster factory, common to any tests (10.x or 4.x)